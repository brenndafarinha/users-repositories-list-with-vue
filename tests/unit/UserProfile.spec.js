import { shallowMount } from '@vue/test-utils';
import UserProfile from '../../src/components/UserProfile';
import store from '../../src/store'

describe('UserProfile', () => {
    it("renders correctly", () => {
        const wrapper = shallowMount(UserProfile, { store });
        expect(wrapper).toMatchSnapshot();
    });
});