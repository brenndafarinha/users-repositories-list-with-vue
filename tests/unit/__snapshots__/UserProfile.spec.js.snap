// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserProfile renders correctly 1`] = `
<v-row>
  <v-col cols="4" class="d-flex justify-between align-center flex-column mx-auto mt-4">
    <v-avatar size="150px"><img alt="Profile image" src=""></v-avatar> <strong class="mt-3"></strong>
  </v-col>
</v-row>
`;
